wasm-wrappers/src/lib.rs

```diff
@@ -587,13 +587,15 @@ pub fn encode_input_for_withdraw_from_delegation(
     Ok(input.encode())
 }
 
-/// Given inputs, outputs and utxos (each encoded as `Option<TxOutput>`), estimate the transaction size.
+/// Given inputs, each input's destination (from the UTXO or Account) and outputs, estimate the transaction size.
```

Comment:
This does not explain how the information should be passed. Please explain it with an example.